test/format/wts.c

```diff
@@ -416,6 +416,21 @@ create_database(const char *home, WT_CONNECTION **connp)
     /* Optional tiered storage. */
     configure_tiered_storage(home, &p, max, tiered_ext_cfg, sizeof(tiered_ext_cfg));
 
+    /* Chunkcache configuration. */
```

Comment:
This should be broken out into a separate function like the other config sections here.

Reply:
Updated in [a07ee2b](https://github.com/wiredtiger/wiredtiger/pull/9589/commits/a07ee2b4200ed8f9d4ff0e5bff69bf29e38be263)